src/vs/editor/contrib/codeAction/browser/codeActionController.ts

```diff
@@ -97,6 +97,10 @@ export class CodeActionController extends Disposable implements IEditorContribut
 		return this.showCodeActionList(actions, at, { includeDisabledActions: false, fromLightbulb: false });
 	}
 
+	public hideCodeActions(): void {
```

Comment:
Looks like these changes got included from another PR?

Reply:
yep, cleaning this up, accidentally brought over that commit